horovod/run/util.py

```diff
@@ -0,0 +1,213 @@
+import paramiko
+import time
+import socket
+import os
+import Queue
+import re
+import sys
+import datetime
+import cloudpickle
+import errno
+import threading
+
+import horovod.spark.util.codec as codec
+
+REMOTE_EXECUTER_SCRIPT = 'horovod.run.exec_script'
+
+
+class Ssh:
+    shell = None
+    client = None
+    transport = None
+
+    def __init__(self, hostname):
+
+        self.client = paramiko.client.SSHClient()
+        self.client.set_missing_host_key_policy(paramiko.client.AutoAddPolicy())
+        try:
+            self.client.connect(hostname, look_for_keys=False)
+        except socket.gaierror:
+            config = paramiko.SSHConfig()
```

Comment:
We probably want to switch back to using stock `ssh` to align how configuration (e.g. hostname mapping) is digested.